exonum/src/blockchain/block.rs

```diff
@@ -119,6 +121,19 @@ pub struct BlockProof {
     pub precommits: Vec<Verified<Precommit>>,
 }
 
+/// Proof of authenticity for a single index within the database.
```

Comment:
I wonder if we implement protobuf serialization for this kind of proof and also by analogy implement `GroupIndexProof`, can we cover all use cases of proofs?